data-plane/receiver/src/main/java/dev/knative/eventing/kafka/broker/receiver/main/Main.java

```diff
@@ -76,9 +66,9 @@
    * @param args command line arguments.
    */
   public static void main(final String[] args) throws IOException {
-    final var env = new ReceiverEnv(System::getenv);
+    ReceiverEnv env = new ReceiverEnv(System::getenv);
 
-    final OpenTelemetrySdk openTelemetry = TracingConfig.fromDir(env.getConfigTracingPath()).setup();
```

Comment:
I actually like `final` to state this is final ...

Reply:
But from Java 8+, this is already implicit when used in closures. Also, in this context, this is bootstrapping code, so obviously everything created here is final and not overwritten :)

Reply:
true - more a nit and pers. pref :-)